extensions/Worker.Extensions.ServiceBus/src/ServiceBusSessionMessageActionsConverter.cs

```diff
@@ -58,5 +54,32 @@ public ValueTask<ConversionResult> ConvertAsync(ConverterContext context)
                 return new ValueTask<ConversionResult>(ConversionResult.Failed(exception));
             }
         }
+
+        private string ParseSessionIdFromBindingData(ConverterContext context)
+        {
+            // Try to resolve sessionId directly
+            var bindingData = context.FunctionContext.BindingContext.BindingData;
+            bindingData.TryGetValue("SessionId", out object? sessionId);
+
+            // If sessionId is not found and sessionIdRepeatedFieldArray has a value (isBatched = true), we can just parse the first sessionId from the array, as all the values are guaranteed to be the same.
+            // This is because there can be multiple messages but each message would belong to the same session.
+            // Note if web jobs extensions ever adds support for multiple sessions in a single batch, this logic will need to be updated.
+            if (sessionId == null && bindingData.TryGetValue("SessionIdArray", out object? sessionIdArray))
```

Comment:
nit: you're using `is` on line 70 so keeping things consistent
```suggestion
            if (sessionId is null && bindingData.TryGetValue("SessionIdArray", out object? sessionIdArray))
```